core/src/main/scala/kafka/log/LogSegment.scala

```diff
@@ -91,17 +98,17 @@ class LogSegment(val log: FileRecords,
    * @param records The log entries to append.
    */
   @nonthreadsafe
-  def append(firstOffset: Long, largestTimestamp: Long, shallowOffsetOfMaxTimestamp: Long, records: MemoryRecords) {
+  def append(firstOffset: Long, largestOffset: Long, largestTimestamp: Long, shallowOffsetOfMaxTimestamp: Long, records: MemoryRecords) {
     if (records.sizeInBytes > 0) {
-      trace("Inserting %d bytes at offset %d at position %d with largest timestamp %d at shallow offset %d"
+      trace("Inserting %d bytes at offset %d at position %d with largest timestamp %d at offset %d"
```

Comment:
I think "shallow offset" is actually correct since we fixed the logic in KAFKA-4390.